docs/phase8-taste-ui-plan.md

```
@@ -0,0 +1,225 @@
+- `test_mcp/test_memory_tools.py` — Add tests for `memory.clear_profile`
+
+**Explorer tests** (`services/explorer/tests/`):
+- `test_routes.py` — Add tests for taste page, update, clear, events partial
+- `conftest.py` — Add mock return values for `get_taste_profile`, `update_taste_profile`, `clear_taste_profile`, `get_preference_events`
+
+---
+
+## Files to Create/Modify
+
+### New Files
+| File | Purpose |
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)